pendulum_control/include/pendulum_control/rtt_executor.hpp

```diff
@@ -120,6 +123,16 @@ class RttExecutor : public rclcpp::Executor
     return 0;
```

Comment:
The `loop_callback` method has to be changed: it does not need the input argument `arg` and can use the `decoratorExecutor` instead.

Reply:
As discussed this comment is no longer relevant.